Source/WebInspectorUI/UserInterface/Views/ConsoleGroup.js

```diff
@@ -79,6 +80,35 @@ WI.ConsoleGroup = class ConsoleGroup extends WI.Object
     {
         this._messagesElement.appendChild(messageOrGroupElement);
     }
+    
+    setMessageLevel(messageLevel)
```

Comment:
Followup: it would be awesome to merge this logic with `WI.ConsoleMessageView.prototype.render` so that we don't have this code duplicated in two places :)

Reply:
This is an excellent point.

Reply:
Added a FIXME with a link to https://webkit.org/b/264489 in 503e2a6cacc0b56c26aaa3bdfcacb6fc6d55e6ae.